applications/osb-portal/src/pages/WorkspacePage.tsx

```diff
@@ -1,25 +1,187 @@
 import * as React from "react";
-import Divider from "@material-ui/core/Divider";
+import { makeStyles } from "@material-ui/core/styles";
+import { useHistory, useParams } from "react-router-dom";
+import MainMenu from "../components/menu/MainMenu";
 import Box from "@material-ui/core/Box";
-import { useParams } from "react-router-dom";
+import Chip from "@material-ui/core/Chip";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import Divider from "@material-ui/core/Divider";
+import AppsIcon from "@material-ui/icons/Apps";
+import PersonIcon from "@material-ui/icons/Person";
+import CalendarTodayIcon from "@material-ui/icons/CalendarToday";
+
+import { OSBSplitButton } from "../components/common/OSBSpliButton";
+import theme, { bgLight, bgRegular, paragraph } from "../theme";
+import WorkspaceService from "../service/WorkspaceService";
+import { Workspace, WorkspaceResource } from "../types/workspace";
+import OSBDialog from "../components/common/OSBDialog";
+import { WorkspaceEditor } from "../components";
+import WorkspaceInteractions from "../components/workspace/drawer/WorkspaceInteractions";
+
+const useStyles = makeStyles(() => ({
+  workspaceToolbar: {
+    backgroundColor: bgLight,
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    cursor: 'pointer',
+    padding: theme.spacing(1),
+    '& .MuiGrid-root': {
+      marginRight: '10px',
+    },
+  },
+  workspaceInformation: {
+    backgroundColor: bgRegular,
+    minHeight: '20vh',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    flexDirection: 'column',
+    padding: theme.spacing(1),
+    '& h1': {
+      fontSize: '24px',
+      fontWeight: '400',
+      marginBottom: theme.spacing(2),
+    },
+    '& .MuiBox-root': {
+      color: paragraph,
+      marginBottom: theme.spacing(2),
+    },
+    '& span': {
+      display: 'flex',
+      alignItems: 'center',
+    },
+    '& .MuiTypography-root:nth-child(2)': {
+      marginLeft: theme.spacing(1),
+    },
+    '& .MuiChip-root': {
+      backgroundColor: '#3c3c3c',
+    },
+  },
+  workspaceResourcesInformation: {
+    display: 'flex',
+    flexDirection: 'row',
+    '& .MuiAccordion-root': {
+      maxWidth: '30vw',
+      borderRadius: 0,
+    },
+  },
+  workspaceDescriptionBox: {
+    width: '70vw',
+    padding: theme.spacing(3),
+    display: 'flex',
+    flexDirection: 'column',
```

Comment:
no need to use css for most of these on Boxes. It's better to use Box properties like alignItems, flex, display, etc